python/doc/examples/calibration/least_squares_and_gaussian_calibration/plot_calibration_chaboche.py

```diff
@@ -1,278 +1,463 @@
+    0
+)
+curve.setLegend("Model before calibration")
+curve.setLineStyle(ot.ResourceMap.GetAsString("CalibrationResult-ObservationLineStyle"))
+graph.add(curve)
+# Plot the noisy observations
+cloud = ot.Cloud(observedStrain, observedStress)
+cloud.setLegend("Observations")
+cloud.setPointStyle(
+    ot.ResourceMap.GetAsString("CalibrationResult-ObservationPointStyle")
+)
+graph.add(cloud)
+graph.setColors(ot.Drawable.BuildDefaultPalette(2))
+graph.setLegendPosition("topleft")
+view = viewer.View(graph)
 
 # %%
-# The `LinearLeastSquaresCalibration` class performs the linear least squares calibration by linearizing the model in the neighbourhood of the reference point.
+# We see that the observations are relatively noisy, but that the
+# trend is clear: this shows that it may be possible to fit the model.
+# At this point, we have a data set that we can use for calibration
+# and a model to calibrate.
+
 
 # %%
+# Calibration with linear least squares
+# -------------------------------------
+# The :class:`~openturns.LinearLeastSquaresCalibration` class performs the
+# linear least squares
+# calibration by linearizing the model in the neighbourhood of the reference point.
 algo = ot.LinearLeastSquaresCalibration(
     mycf, observedStrain, observedStress, thetaPrior, "SVD"
 )
 
 # %%
-# The `run` method computes the solution of the problem.
-
-# %%
+# The :meth:`~openturns.LinearLeastSquaresCalibration.run` method computes
+# the solution of the problem.
 algo.run()
-
-# %%
 calibrationResult = algo.getResult()
 
 # %%
 # Analysis of the results
 # -----------------------
-
-# %%
-# The `getParameterMAP` method returns the maximum of the posterior distribution of :math:`\theta`.
-
-# %%
+# The :meth:`~openturns.CalibrationResult.getParameterMAP` method
+# returns the maximum of the posterior distribution of :math:`\theta`.
 thetaMAP = calibrationResult.getParameterMAP()
-thetaMAP
+print("theta After = ")
+print("    R = %.2f (MPa)" % (thetaMAP[0] / 1.0e6))
+print("    C = %.2f (MPa)" % (thetaMAP[1] / 1.0e6))
+print("    Gamma = %.4f" % (thetaMAP[2]))
+print("theta Before = ")
+print("    R = %.2f (MPa)" % (thetaPrior[0] / 1.0e6))
+print("    C = %.2f (MPa)" % (thetaPrior[1] / 1.0e6))
+print("    Gamma = %.4f" % (thetaPrior[2]))
+print("theta True = ")
+print("    R = %.2f (MPa)" % (cm.trueR / 1.0e6))
+print("    C = %.2f (MPa)" % (cm.trueC / 1.0e6))
+print("    Gamma = %.4f" % (cm.trueGamma))
 
 # %%
 # We can compute a 95% confidence interval of the parameter :math:`\theta^\star`.
-
-# %%
 thetaPosterior = calibrationResult.getParameterPosterior()
 thetaPosterior.computeBilateralConfidenceIntervalWithMarginalProbability(0.95)[0]
 
 # %%
-# We can see that the `Gamma` parameter has a large confidence interval : even the sign of the parameter is unknown.
+# We can see that the `Gamma` parameter has a large confidence interval:
+# even the sign of the parameter is unknown.
 # The parameter which is calibrated with the smallest confidence
```

Comment:
Putting a comment here to be able to link to https://github.com/openturns/openturns/pull/2060#discussion_r1166966383 which is unresolved IMO.

Reply:
Sorry for resolving the issue without your consent. Please tell me when OK.

Reply:
It is OK (see my answer to the thread I linked to). Thanks!